core/src/com/ericc/the/game/mapGenerator/Coordinates.java

```diff
@@ -0,0 +1,10 @@
+package com.ericc.the.game.mapGenerator;
+
+public class Coordinates<T> {
```

Comment:
Nie wiem w sumie czy jest sens żeby ta klasa była generyczna, a poza tym to znalazłem analogiczną w api libgdxa https://libgdx.badlogicgames.com/ci/nightlies/docs/api/com/badlogic/gdx/math/GridPoint2.html